pkgs/applications/audio/denemo/default.nix

```diff
@@ -2,7 +2,7 @@
 , libjack2, gettext, intltool, guile_2_0, lilypond
 , glib, libxml2, librsvg, libsndfile, aubio
 , gtk3, gtksourceview, evince, fluidsynth, rubberband
-, portaudio, portmidi, fftw, makeWrapper }:
+, portaudio, portmidi, fftw, makeWrapper, wrapGAppsHook }:
```

Comment:
No need to use wrapProgram, you can remove one wrapping layer using `gappsWrapperArgs` as described in https://nixos.org/nixpkgs/manual/#ssec-gnome-hooks.

Reply:
fixed.